lib/generators/jsonapi/templates/destroy_request_spec.rb.erb

```diff
@@ -2,7 +2,7 @@ require 'rails_helper'
 
 RSpec.describe "<%= type %>#destroy", type: :request do
   context 'basic destroy' do
-    let!(:<%= file_name %>) { FactoryGirl.create(:<%= file_name %>) }
+    let!(:<%= file_name %>) { FactoryBot.create(:<%= file_name %>) }
```

Comment:
You can actually change this to just `create`, it's a holdover of when I was originally not mixin in `FactoryGirl::Syntax::Methods`

Reply:
ah forgot we'd added the methods mixin. will update.